cap-ci/jobs/deploy-k8s.tmpl

```diff
@@ -2,13 +2,17 @@
 - name: deploy-k8s-{{ .scheduler }}-{{ .backend }}-{{ .avail }}
   public: false
   plan:
+  {{- if .schedule }}
+  - get: schedule-interval
+    trigger: true
+  {{ end }}
   - get: s3.kubecf-bundle
+  {{- if not .schedule }}
     trigger: true
+  {{ end }}
   {{- if index .jobs "deploy-stratos" }}
   - get: helm-chart.stratos-chart
-    trigger: true
   - get: helm-chart.stratos-metrics-chart
-    trigger: true
```

Comment:
we still need these triggers for CI using pet clusters

Reply:
same as above.